### PR TITLE
Feature/enable psc filing rejection

### PR DESCRIPTION
**Pull request by clewis1**
- Added PscAcceptanceStrategy

### CODE REVIEW

**Comment by clewis1** [comment]
https://companieshouse.atlassian.net/browse/PSC-64

File: src/main/java/uk/gov/companieshouse/filingmock/processor/strategy/AcceptanceStrategyFactory.java [hunk 1]
```diff
@@ -22,6 +23,8 @@ public static AcceptanceStrategy getStrategy(Transaction submission) {
             // There are multiple insolvency types e.g. insolvency#600 but all will start with "insolvency," and should use the same strategy
         } else if (submissionType.contains("insolvency")) {
             return INSOLVENCY;
+        } else if (submissionType.contains("cessation")) {
+            return CESSATION;
```
Comment (hepsimo) [inline src/main/java/uk/gov/companieshouse/filingmock/processor/strategy/AcceptanceStrategyFactory.java:27]: We'll need test coverage for this in `AcceptanceStrategyFactoryTest`

**Review by hepsimo — CHANGES_REQUESTED** [review]
Changes look good, just a bit more test coverage needed.

**Review by hepsimo — APPROVED** [review]
Changes look good

File: src/main/java/uk/gov/companieshouse/filingmock/processor/strategy/AcceptanceStrategyFactory.java (hunk 1 above)
Comment (clewis1) [reply src/main/java/uk/gov/companieshouse/filingmock/processor/strategy/AcceptanceStrategyFactory.java:27]: Changes applied

**Pull request merged**